scripts/start.sh

```diff
@@ -77,7 +77,7 @@ docker-compose up -d schemaregistry connect control-center
 
 echo
 echo -e "Create topics in Kafka cluster:"
-docker-compose exec kafka1 bash -c "/tmp/helper/create-topics.sh" || exit 1
+docker-compose exec tools bash -c "/tmp/helper/create-topics.sh" || exit 1
```

Comment:
@javabrett can you please remind me, why this is executed on the `tools` container (which might change in the future) instead of `kafka1`?

Reply:
I accidentally resolved a conversation - https://github.com/confluentinc/cp-demo/pull/300#discussion_r546328817 and also see https://github.com/confluentinc/cp-demo/pull/300#issuecomment-748557592 .

The new scripting requires `curl` and `jq`, as does the suggested `get_kafka_cluster_id_from_container`, which also runs on `tools`, since it is the only container that has both.

I think we might be best to defer solving this until we know what is available for the `tools` replacement - i.e. we might need a local utils image, or the new CLI image might provide.

If we can't run on `tools` we would also back-out the use `get_kafka_cluster_id_from_container` refactor, which needs to run on `tools`.

Reply:
Agree, let's leave it on `tools` for now.